src/WebJobs.Extensions.DurableTask/Listener/DurableTaskTargetScaler.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if FUNCTIONS_V3_OR_GREATER
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Castle.Components.DictionaryAdapter;
+using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Listener
+{
+    internal class DurableTaskTargetScaler : ITargetScaler
+    {
+        private readonly DurableTaskMetricsProvider metricsProvider;
+        private readonly TargetScalerResult scaleResult;
+        private readonly DurabilityProvider durabilityProvider;
+
+        public DurableTaskTargetScaler(string functionId, DurableTaskMetricsProvider metricsProvider, DurabilityProvider durabilityProvider)
+        {
+            this.metricsProvider = metricsProvider;
+            this.scaleResult = new TargetScalerResult();
+            this.TargetScalerDescriptor = new TargetScalerDescriptor(functionId);
```

Comment:
I see that `functionId` is used to create the `TargetScalerDescriptor`. Are there any other uses for `functionId`?

Reply:
I don't think so. I believe this is literally just the function name as a string. It's called `functionId` for convention, but it might as well be `functionName`.